public/app/features/alerting/unified/components/settings/InternalAlertmanager.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { ConnectionStatus } from '../../hooks/useExternalAmSelector';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
+import { isReceivingOnInternalAlertmanager } from '../../utils/settings';
+
+import { AlertmanagerCard } from './AlertmanagerCard';
+import { useSettings } from './SettingsContext';
+
+interface Props {
+  onEditConfiguration: (dataSourceName: string) => void;
+}
+
+export default function InternalAlertmanager({ onEditConfiguration }: Props) {
```

Comment:
I noticed that while in the UI we updated this naming to use Built-in instead of internal, in the code we are using the old naming (internal/external). Maybe not for this PR but we could update this naming in the code?